cmake/Caffe2Config.cmake.in

```diff
@@ -118,7 +118,6 @@ include ("${CMAKE_CURRENT_LIST_DIR}/Caffe2Targets.cmake")
 # Interface libraries, that allows one to build proper link flags.
 # We will also define a helper variable, Caffe2_MAIN_LIBS, that resolves to
 # the main caffe2 libraries in cases of cuda presence / absence.
-caffe2_interface_library(torch torch_library)
```

Comment:
I'm not familiar with caffe2 stuff and why it was here in the first place... seems redundant?

Reply:
Previously, `torch` was an exported target but `torch_library` was not. So the Config.cmake (which is what gets loaded up when people find pytorch) needed to "make" the library again. So it's not redundant. However, what I don't understand is why we didn't just export `torch_library` in the first place. I wasn't able to figure this out (though, admittedly, I didn't try very hard.)